LICENSES/LGPL-3.0-or-later.txt

```diff
@@ -0,0 +1,55 @@
+GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
why LGPL? i prefer GPL.

Reply:
Hi Stefan, 
in the file `pwdsphinx/sphinxlib.py ` you indicate that the license is LGPL:
pitchforked sphinx is free software: you can redistribute it and/or modify it under the terms of the **GNU Lesser General Public License** as published by the Free Software Foundation
If it is not the case, and you will put the code under GPL v3, i can edit the pull request

Reply:
oh, that is my mistake then. i'm quite sure i wanted gpl.

Reply:
Changed license information to GPL v3 in all respective files